remoto/tests/test_process.py

```diff
@@ -1,6 +1,7 @@
 from mock import Mock
+from mock import patch
 from remoto import process
-
+import subprocess
```

Comment:
is this import required because of the patch?

Reply:
Yes